packages/genesys-spark-components/src/components/beta/gux-side-sheet/example.html

```diff
@@ -0,0 +1,73 @@
+<div class="side-sheet-content">
+  <h1>Side Sheet</h1>
+</div>
+
+<gux-side-sheet-beta size="medium">
```

Comment:
I think you should include examples of `small` and `large` also.

Reply:
@gavin-everett-genesys yes, 100%, just looking for API feedback at the moment.